src/resources.rs

```diff
@@ -69,14 +70,21 @@ fn fetch_http(url: &Url) -> Vec<u8> {
     }
 }
 
+#[cfg(not(feature = "ureq"))]
+#[allow(unreachable_code)]
```

Comment:
Why's this?  What's unreachable here?

Reply:
Ah, sorry, I should add a comment. See https://github.com/withoutboats/fehler/issues/42

Reply:
I wonder whether it'd be simpler to just use `anyhow::Result<Vec<u8>>` as return type and use `Err(anyhow!(…))` as body… after all we don't really need the early return feature from fehler in this one, do we?

Reply:
If I do this, I will also have to add conditional imports. Otherwise it complains (and errors out):

```
error: unused import: `Error`
 --> src/resources.rs:9:31
  |
9 | use anyhow::{anyhow, Context, Error, Result};
  |                               ^^^^^
  |
note: the lint level is defined here
 --> src/lib.rs:7:9
  |
7 | #![deny(warnings, missing_docs, clippy::all)]
  |         ^^^^^^^^
  = note: `#[deny(unused_imports)]` implied by `#[deny(warnings)]`

error: unused import: `throws`
  --> src/resources.rs:10:21
   |
10 | use fehler::{throw, throws};
   |                     ^^^^^^
```

Reply:
Didn't think of that.  Leave it as is then, just do me a favour and add a comment which references the fehler issue so that I don't forget why the unreachable thing is there 🙂